### DISCUSSION TITLE
Outside collaborators

### DISCUSSION THREAD

**Discussion by Icalix** [community/community#38698] [Repositories]
### Select Topic Area

General

### Body

Hi, at our organization we have several projects, I need to invite an outside collaborator, that process I know how to, however, I want to limit access for him to the one project he will be involved in and prevent access for the rest of the projects.

Thanks

**Comment by ossanna16** [comment]
Hi there @Icalix and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
